src/libraries/System.Linq.Expressions/src/System/Linq/IQueryable.cs

```diff
@@ -7,99 +7,105 @@
+    /// <remarks>The <see cref="System.Linq.IQueryable" /> interface is intended for implementation by query providers. It is only supposed to be implemented by providers that also implement <see cref="System.Linq.IQueryable{T}" />. If the provider does not also implement <see cref="System.Linq.IQueryable{T}" />, the standard query operators cannot be used on the provider's data source.
+    /// The <see cref="System.Linq.IQueryable" /> interface inherits the <see cref="System.Collections.IEnumerable" /> interface so that if it represents a query, the results of that query can be enumerated. Enumeration causes the expression tree associated with an <see cref="System.Linq.IQueryable" /> object to be executed. The definition of "executing an expression tree" is specific to a query provider. For example, it may involve translating the expression tree to an appropriate query language for the underlying data source. Queries that do not return enumerable results are executed when the <see cref="O:System.Linq.IQueryProvider.Execute" /> method is called.
+    /// For more information about how to create your own LINQ provider, see <a href="https://docs.microsoft.com/archive/blogs/mattwar/linq-building-an-iqueryable-provider-part-i">LINQ: Building an IQueryable Provider</a>.</remarks>
+    /// <altmember langword="System.Linq.Queryable"/>
+    /// <altmember cref="System.Linq.IQueryable{T}"/>
     public interface IQueryable : IEnumerable
     {
-        /// <summary>
-        /// Gets the expression tree that is associated with the instance of <see cref="IQueryable"/>.
-        /// </summary>
+        /// <summary>Gets the expression tree that is associated with the instance of <see cref="System.Linq.IQueryable" />.</summary>
+        /// <value>The <see cref="System.Linq.Expressions.Expression" /> that is associated with this instance of <see cref="System.Linq.IQueryable" />.</value>
+        /// <remarks>If an instance of <see cref="System.Linq.IQueryable" /> represents a LINQ query against a data source, the associated expression tree represents that query.</remarks>
         Expression Expression { get; }
 
-        /// <summary>
-        /// Gets the type of the element(s) that are returned when the expression tree associated with this instance of <see cref="IQueryable"/> is executed.
-        /// </summary>
+        /// <summary>Gets the type of the element(s) that are returned when the expression tree associated with this instance of <see cref="System.Linq.IQueryable" /> is executed.</summary>
+        /// <value>A <see cref="System.Type" /> that represents the type of the element(s) that are returned when the expression tree associated with this object is executed.</value>
+        /// <remarks>The <see cref="O:System.Linq.IQueryable.ElementType" /> property represents the "T" in `IQueryable&lt;T&gt;` or `IQueryable(Of T)`.</remarks>
```

Comment:
To use the backticks here, it would need to be in a markdown CDATA section. Otherwise use `<code>`.